brainframe_qt/ui/dialogs/task_configuration/widgets/video_task_config.py

```diff
@@ -0,0 +1,136 @@
+from typing import List, Optional
```

Comment:
Is this all new, or from another file?

Reply:
A bit of both. The file was moved and changed, and that confused git enough

Reply:
Which file was it from? I can manually diff it on my end to try to provide feedback

Reply:
`brainframe_qt/ui/dialogs/task_configuration/video_task_config/video_task_config.py`